api/capi/src/nnstreamer-capi-single.c

```diff
@@ -615,7 +615,7 @@ ml_single_get_tensors_info (ml_single_h single, gboolean is_input,
 }
 
 /**
- * @brief Gets the type (tensor dimension, type, name and so on) of required input data for the given handle.
+ * @brief Gets the information (tensor dimension, type, name and so on) of required input data for the given handle.
```

Comment:
Q. the given handle == the given model?

Reply:
Yes, as this is single API, one handle corresponds to one (single) model.